resources/public/api/conf/1.0/employment_expenses_delete.yaml

```diff
@@ -0,0 +1,83 @@
+summary: Delete Employment Expenses
+description: |
+  This endpoint allows a developer to delete employment expenses for a user and can only be used after the tax year has ended.<br>Deleting an expense will remove the amended expense and remove the "dateIgnored" flag from HMRC expenses.
```

Comment:
According to the spec this description should be `This endpoint allows a developer to delete employment expenses for a user. This endpoint can only be used after the tax year has ended. A National Insurance number and Tax year are required.

Deleting an expense will remove the amended expense and remove the "dateIgnored" flag from HMRC expenses.`